### PR TITLE
Fix #17342 - Fix indentation / markdown formatting for list numbering.

### LINKED ISSUE #17342: Issue with k8s.io/docs/tasks/administer-cluster/namespaces/
**This is a Bug Report**

**Problem:**
Ordered list is broken. Each item starts "1".
> 1 . Understand the default namespace
> ...
> 1 . Create new namespaces
> ...
> 1 . Create pods in each namespace

https://kubernetes.io/docs/tasks/administer-cluster/namespaces/#subdividing-your-cluster-using-kubernetes-namespaces

**Proposed Solution:**
> 1. Understand the default namespace
> ...
> 1. Create new namespaces
> ...
> 1. Create pods in each namespace

**Page to Update:**
https://kubernetes.io/docs/tasks/administer-cluster/namespaces/

### PR DESCRIPTION
**Pull request by miteshskj**
fixes #17342

### CODE REVIEW

**Comment by miteshskj** [comment]
/assign @sftim 
redux - https://github.com/kubernetes/website/pull/17348

File: content/en/docs/tasks/administer-cluster/namespaces.md [hunk 1]
```diff
@@ -125,199 +125,195 @@ This delete is asynchronous, so for a time you will see the namespace in the `Te
...
+    ```
 
-You can verify your current context by doing the following:
+    You can verify your current context by doing the following:
 
-```shell
-kubectl config current-context
-dev
-```
+    ```shell
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/namespaces.md]: This isn't quite right (it mixes the command with its output), _but_ the fix should be a separate PR.
